DXRMapFixups/DeusEx/Classes/DXRFixupIntro.uc

```diff
@@ -11,6 +11,13 @@ function PreFirstEntryMapFixes()
         foreach AllActors(class'#var(DeusExPrefix)Mover',dxm,'trap_door'){
             DestroyMover(dxm);
         }
+
+        if (dxr.flags.moresettings.empty_medbots > 0) {
+            class'AugmentationCannister'.default.MustBeUsedOn = "Can only be installed with the help of a MedBot or AugBot.";
+        } else {
+            class'AugmentationCannister'.default.MustBeUsedOn = "Can only be installed with the help of a MedBot.";
```

Comment:
this doesn't work if you save and quit, in order to work for all mods it would probably need a `foreach AllActors` loop inside of DXRFixups.uc AnyEntry

Reply:
if you move the setting the defaults into AnyEntry, it works without a foreach loop, but actually because it prepares the default in time for the next map (in the case of new games, setting it in the intro is what gets the default ready for liberty island), and according to the t3d files none of the cans placed in the maps will override MustBeUsedOn